helm/rtd/values.yaml

```diff
@@ -63,7 +63,6 @@ microservice-chart:
 
   envConfigMapExternals:
     rtd-fileregister:
-      JAVA_TOOL_OPTIONS: JAVA_TOOL_OPTIONS
       APPLICATIONINSIGHTS_ROLE_NAME: APPLICATIONINSIGHTS_ROLE_NAME
       APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL: APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL
```

Comment:
appinsights instrumentation variables are still needed?

Reply:
`APPLICATIONINSIGHTS_ROLE_NAME` and `APPLICATIONINSIGHTS_INSTRUMENTATION_MICROMETER_ENABLED` are no longer used. 

While `APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL` is used as alias for OPENTELEMETRY_LOG_LEVEL. 

We can map it in the following way: 
`OPENTELEMETRY_LOG_LEVEL: APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL`